src/utils/tasks.ts

```diff
@@ -50,3 +51,16 @@ export const getFlatArrayFromObjectValues = (survey: any): Task[] => {
   )
   return arrayFlatTasks
 }
+
+export const getUnansweredGuidelines = (survey: any, auditProgram: any): any => {
+  const answeredGuide = Object.values(survey.answers)
+  const totalGuidelines = Object.values(auditProgram.guidelines)
+  const pkOfAnsweredGuide = answeredGuide.map((answer: any) => answer.guideline_pk)
```

Comment:
This could be called just ```answerPks```. You are llooping an array of ```Answer``` so that should be the type instead of ```any```